Synthesis failed for nodejs-logging

Hello! Autosynth couldn't regenerate nodejs-logging. :broken_heart:

Here's the output from running `synth.py`:

```
ool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2020-12-23 04:26:04,108 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpmyqb9dac.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpmyqb9dac.
2020-12-23 04:26:04,134 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2020-12-23 04:26:04,262 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-12-23 04:26:04,262 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN cheerio-select-tmp@0.1.0 requires a peer of @types/node@^14.11.2 but none is installed. You must install peer dependencies yourself.

added 776 packages from 825 contributors and audited 789 packages in 32.558s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-23 04:26:37,482 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN cheerio-select-tmp@0.1.0 requires a peer of @types/node@^14.11.2 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 789 packages in 19.526s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN cheerio-select-tmp@0.1.0 requires a peer of @types/node@^14.11.2 but none is installed. You must install peer dependencies yourself.

added 269 packages from 174 contributors and audited 396 packages in 7.017s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-23 04:27:05,672 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2020-12-23 04:27:48,425 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2020-12-23 04:27:54,263 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2020-12-23 04:27:56,307 autosynth [ERROR] > Synthesis failed
2020-12-23 04:27:56,307 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 47b050e docs: add instructions for authenticating for system tests (#974)
2020-12-23 04:27:56,335 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-12-23 04:27:56,362 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/56d7ac22-6f07-4b7e-95d7-32d95ffe5b62/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
n
2020-12-24 04:25:14,252 synthtool [DEBUG] > Ensuring dependencies.
DEBUG:synthtool:Ensuring dependencies.
2020-12-24 04:25:14,261 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-12-24 04:25:14,262 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-12-24 04:25:14,266 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2020-12-24 04:25:19,531 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpgmbwaz1v.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpgmbwaz1v.
2020-12-24 04:25:19,566 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2020-12-24 04:25:19,683 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-12-24 04:25:19,683 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 776 packages from 824 contributors and audited 789 packages in 29.762s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-24 04:25:50,061 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 789 packages in 17.574s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 269 packages from 174 contributors and audited 396 packages in 6.555s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-24 04:26:15,787 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2020-12-24 04:26:54,141 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2020-12-24 04:26:59,465 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2020-12-24 04:27:01,507 autosynth [ERROR] > Synthesis failed
2020-12-24 04:27:01,507 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 47b050e docs: add instructions for authenticating for system tests (#974)
2020-12-24 04:27:01,535 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-12-24 04:27:01,559 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/0855e0cd-61ba-4f1a-b302-6b6d4a84b649/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
n
2020-12-25 04:27:06,843 synthtool [DEBUG] > Ensuring dependencies.
DEBUG:synthtool:Ensuring dependencies.
2020-12-25 04:27:06,853 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-12-25 04:27:06,854 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-12-25 04:27:06,859 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2020-12-25 04:27:12,562 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp34s4b9y0.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp34s4b9y0.
2020-12-25 04:27:12,585 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2020-12-25 04:27:12,719 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-12-25 04:27:12,719 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 776 packages from 824 contributors and audited 789 packages in 34.219s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-25 04:27:47,667 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 789 packages in 20.423s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 269 packages from 174 contributors and audited 396 packages in 7.431s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-25 04:28:17,242 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2020-12-25 04:29:00,833 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2020-12-25 04:29:06,850 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2020-12-25 04:29:08,900 autosynth [ERROR] > Synthesis failed
2020-12-25 04:29:08,900 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 47b050e docs: add instructions for authenticating for system tests (#974)
2020-12-25 04:29:08,933 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-12-25 04:29:08,960 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/20d348c5-6027-41be-b620-7c3ed299ebb5/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
G:synthtool:Ensuring dependencies.
2020-12-26 04:26:48,208 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-12-26 04:26:48,209 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-12-26 04:26:48,214 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2020-12-26 04:26:54,092 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp4dfier43.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp4dfier43.
2020-12-26 04:26:54,114 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2020-12-26 04:26:54,248 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-12-26 04:26:54,248 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 765 packages from 825 contributors and audited 778 packages in 34.635s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-26 04:27:29,589 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 778 packages in 20.312s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 269 packages from 174 contributors and audited 396 packages in 7.653s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-26 04:27:59,330 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2020-12-26 04:28:43,811 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2020-12-26 04:28:49,914 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2020-12-26 04:28:51,960 autosynth [ERROR] > Synthesis failed
2020-12-26 04:28:51,961 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 47b050e docs: add instructions for authenticating for system tests (#974)
2020-12-26 04:28:51,993 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-12-26 04:28:52,020 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/8da34550-d28e-41ab-abf8-edc158fd0376/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
G:synthtool:Ensuring dependencies.
2020-12-27 04:26:42,308 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-12-27 04:26:42,309 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-12-27 04:26:42,313 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2020-12-27 04:26:48,481 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpa3kimzf2.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpa3kimzf2.
2020-12-27 04:26:48,503 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2020-12-27 04:26:48,635 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-12-27 04:26:48,635 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 765 packages from 825 contributors and audited 778 packages in 34.158s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-27 04:27:23,497 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 778 packages in 20.103s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 269 packages from 174 contributors and audited 396 packages in 7.455s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-27 04:27:52,782 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2020-12-27 04:28:36,771 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2020-12-27 04:28:42,936 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2020-12-27 04:28:44,985 autosynth [ERROR] > Synthesis failed
2020-12-27 04:28:44,986 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 47b050e docs: add instructions for authenticating for system tests (#974)
2020-12-27 04:28:45,017 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-12-27 04:28:45,039 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/50ed7667-f4e4-4750-b1dc-fa866ec414a9/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
BUG:synthtool:Ensuring dependencies.
2020-12-28 04:27:09,525 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-12-28 04:27:09,526 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-12-28 04:27:09,531 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2020-12-28 04:27:15,679 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp0lrr32om.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp0lrr32om.
2020-12-28 04:27:15,712 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2020-12-28 04:27:15,851 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-12-28 04:27:15,852 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 765 packages from 825 contributors and audited 778 packages in 35.792s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-28 04:27:52,382 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 778 packages in 20.64s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 269 packages from 174 contributors and audited 396 packages in 7.75s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-28 04:28:22,542 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2020-12-28 04:29:06,848 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2020-12-28 04:29:13,086 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2020-12-28 04:29:15,141 autosynth [ERROR] > Synthesis failed
2020-12-28 04:29:15,142 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 47b050e docs: add instructions for authenticating for system tests (#974)
2020-12-28 04:29:15,174 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-12-28 04:29:15,201 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/090f0d77-c67e-4aab-89f6-040a9bfea294/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
ependencies.
DEBUG:synthtool:Ensuring dependencies.
2020-12-29 04:28:19,578 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-12-29 04:28:19,580 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-12-29 04:28:19,585 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2020-12-29 04:28:25,652 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpdp3cbr8j.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpdp3cbr8j.
2020-12-29 04:28:25,684 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2020-12-29 04:28:25,826 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-12-29 04:28:25,827 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 765 packages from 825 contributors and audited 778 packages in 37.39s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-29 04:29:03,977 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 778 packages in 21.378s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 269 packages from 174 contributors and audited 396 packages in 8.078s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-29 04:29:35,271 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2020-12-29 04:30:20,516 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2020-12-29 04:30:26,931 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2020-12-29 04:30:28,991 autosynth [ERROR] > Synthesis failed
2020-12-29 04:30:28,991 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bdb71e4 ci: fix bucket_not_found for samples tests (#978)
2020-12-29 04:30:29,022 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-12-29 04:30:29,045 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/e2e24334-87c7-4d83-a5f1-63176663c84a/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
ependencies.
DEBUG:synthtool:Ensuring dependencies.
2020-12-30 04:25:58,445 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-12-30 04:25:58,445 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-12-30 04:25:58,450 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2020-12-30 04:26:04,319 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp3r729j4q.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp3r729j4q.
2020-12-30 04:26:04,341 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2020-12-30 04:26:04,474 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-12-30 04:26:04,474 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 753 packages from 823 contributors and audited 766 packages in 32.23s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-30 04:26:37,365 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 766 packages in 19.406s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 269 packages from 174 contributors and audited 396 packages in 7.043s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-30 04:27:05,468 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2020-12-30 04:27:47,879 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2020-12-30 04:27:53,655 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2020-12-30 04:27:55,699 autosynth [ERROR] > Synthesis failed
2020-12-30 04:27:55,699 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bdb71e4 ci: fix bucket_not_found for samples tests (#978)
2020-12-30 04:27:55,729 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-12-30 04:27:55,755 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/ffd4cff5-0257-40ae-84d1-8fc73ad646fe/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
pendencies.
DEBUG:synthtool:Ensuring dependencies.
2020-12-31 04:27:45,082 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-12-31 04:27:45,083 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-12-31 04:27:45,087 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2020-12-31 04:27:51,078 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpp8taxs4y.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpp8taxs4y.
2020-12-31 04:27:51,116 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2020-12-31 04:27:51,291 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-12-31 04:27:51,291 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 753 packages from 823 contributors and audited 766 packages in 36.207s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-31 04:28:28,199 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 766 packages in 20.661s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 269 packages from 174 contributors and audited 396 packages in 7.833s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2020-12-31 04:28:58,484 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2020-12-31 04:29:43,684 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2020-12-31 04:29:49,990 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2020-12-31 04:29:52,043 autosynth [ERROR] > Synthesis failed
2020-12-31 04:29:52,044 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bdb71e4 ci: fix bucket_not_found for samples tests (#978)
2020-12-31 04:29:52,075 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-12-31 04:29:52,102 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/340e5135-8aab-4c3f-ba80-8dfd42c3e995/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
pendencies.
DEBUG:synthtool:Ensuring dependencies.
2021-01-01 04:55:26,808 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2021-01-01 04:55:26,809 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-01 04:55:26,814 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2021-01-01 04:55:56,650 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp6jcawv43.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp6jcawv43.
2021-01-01 04:55:56,672 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2021-01-01 04:55:56,804 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2021-01-01 04:55:56,804 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 750 packages from 823 contributors and audited 763 packages in 33.665s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-01 04:56:31,160 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 763 packages in 19.438s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 269 packages from 174 contributors and audited 396 packages in 7.286s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-01 04:56:59,541 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2021-01-01 04:57:42,909 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2021-01-01 04:57:48,870 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2021-01-01 04:57:50,917 autosynth [ERROR] > Synthesis failed
2021-01-01 04:57:50,917 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bdb71e4 ci: fix bucket_not_found for samples tests (#978)
2021-01-01 04:57:50,948 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-01 04:57:50,974 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/29069528-f186-4ec9-92b5-d225fd8530ec/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
pendencies.
DEBUG:synthtool:Ensuring dependencies.
2021-01-02 04:38:56,425 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2021-01-02 04:38:56,426 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-02 04:38:56,431 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2021-01-02 04:39:01,294 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp3nc76qx4.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp3nc76qx4.
2021-01-02 04:39:01,322 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2021-01-02 04:39:01,458 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2021-01-02 04:39:01,458 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 749 packages from 823 contributors and audited 762 packages in 35.301s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-02 04:39:37,453 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 762 packages in 20.184s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 268 packages from 174 contributors and audited 395 packages in 7.681s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-02 04:40:07,036 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2021-01-02 04:40:51,022 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2021-01-02 04:40:57,070 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2021-01-02 04:40:59,121 autosynth [ERROR] > Synthesis failed
2021-01-02 04:40:59,121 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bdb71e4 ci: fix bucket_not_found for samples tests (#978)
2021-01-02 04:40:59,153 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-02 04:40:59,181 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/99731fff-7c75-46cb-812c-f63454d31623/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
pendencies.
DEBUG:synthtool:Ensuring dependencies.
2021-01-03 04:41:31,120 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2021-01-03 04:41:31,122 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-03 04:41:31,127 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2021-01-03 04:41:36,570 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpmz5mtu4c.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpmz5mtu4c.
2021-01-03 04:41:36,593 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2021-01-03 04:41:36,731 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2021-01-03 04:41:36,732 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 749 packages from 823 contributors and audited 762 packages in 37.224s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-03 04:42:14,707 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 762 packages in 21.255s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 268 packages from 174 contributors and audited 395 packages in 8.454s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-03 04:42:46,270 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2021-01-03 04:43:31,848 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2021-01-03 04:43:38,374 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2021-01-03 04:43:40,441 autosynth [ERROR] > Synthesis failed
2021-01-03 04:43:40,441 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bdb71e4 ci: fix bucket_not_found for samples tests (#978)
2021-01-03 04:43:40,478 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-03 04:43:40,506 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/6ef485f6-2d6d-4e57-a4e5-78143e77cb9c/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
pendencies.
DEBUG:synthtool:Ensuring dependencies.
2021-01-04 04:38:10,891 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2021-01-04 04:38:10,891 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-04 04:38:10,896 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2021-01-04 04:38:15,813 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmps68p94bj.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmps68p94bj.
2021-01-04 04:38:15,836 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2021-01-04 04:38:15,968 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2021-01-04 04:38:15,968 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 749 packages from 823 contributors and audited 762 packages in 32.866s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-04 04:38:49,510 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 762 packages in 19.521s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.3/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 268 packages from 174 contributors and audited 395 packages in 7.131s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-04 04:39:17,811 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2021-01-04 04:40:01,090 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2021-01-04 04:40:06,986 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2021-01-04 04:40:09,030 autosynth [ERROR] > Synthesis failed
2021-01-04 04:40:09,030 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bdb71e4 ci: fix bucket_not_found for samples tests (#978)
2021-01-04 04:40:09,059 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-04 04:40:09,086 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/a35d6a53-53a0-494c-af27-b5513784d260/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
h.py.
On branch autosynth-86
nothing to commit, working tree clean
2021-01-05 04:39:54,044 synthtool [DEBUG] > Ensuring dependencies.
DEBUG:synthtool:Ensuring dependencies.
2021-01-05 04:39:54,054 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2021-01-05 04:39:54,055 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-05 04:39:54,060 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2021-01-05 04:39:59,123 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpw6x759ea.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpw6x759ea.
2021-01-05 04:39:59,145 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2021-01-05 04:39:59,281 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2021-01-05 04:39:59,281 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 749 packages from 823 contributors and audited 762 packages in 36.147s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-05 04:40:36,237 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 762 packages in 20.503s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.4/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.4/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 268 packages from 174 contributors and audited 395 packages in 7.774s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-05 04:41:06,257 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2021-01-05 04:41:50,973 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2021-01-05 04:41:57,219 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2021-01-05 04:41:59,276 autosynth [ERROR] > Synthesis failed
2021-01-05 04:41:59,276 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bdb71e4 ci: fix bucket_not_found for samples tests (#978)
2021-01-05 04:41:59,307 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-05 04:41:59,334 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d2e70c3f-ef7f-48e1-9345-0af584a5176a/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
Ensuring dependencies.
2021-01-06 04:42:45,305 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2021-01-06 04:42:45,305 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-06 04:42:45,309 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2021-01-06 04:43:04,596 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmppxiqdgeh.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmppxiqdgeh.
2021-01-06 04:43:04,615 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2021-01-06 04:43:04,728 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2021-01-06 04:43:04,728 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead
npm WARN deprecated fsevents@2.1.3: "Please update to latest v2.3 or v2.2"

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 728 packages from 823 contributors and audited 741 packages in 27.883s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-06 04:43:33,159 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 741 packages in 16.744s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.4/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.4/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: "Please update to latest v2.3 or v2.2"
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 268 packages from 174 contributors and audited 368 packages in 6.029s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-06 04:43:57,392 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2021-01-06 04:44:34,176 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2021-01-06 04:44:39,247 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2021-01-06 04:44:41,286 autosynth [ERROR] > Synthesis failed
2021-01-06 04:44:41,286 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bdb71e4 ci: fix bucket_not_found for samples tests (#978)
2021-01-06 04:44:41,311 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-06 04:44:41,333 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/a85635cc-8674-4c51-be75-5029f85b88cb/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
l:Ensuring dependencies.
2021-01-07 04:38:21,711 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2021-01-07 04:38:21,712 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-07 04:38:21,716 synthtool [DEBUG] > Generating code for: //google/logging/v2:logging-v2-nodejs.
DEBUG:synthtool:Generating code for: //google/logging/v2:logging-v2-nodejs.
2021-01-07 04:38:26,669 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpnruxgc2k.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpnruxgc2k.
2021-01-07 04:38:26,692 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.kokoro/trampoline_v2.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
.trampolinerc
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
2021-01-07 04:38:26,823 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2021-01-07 04:38:26,823 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: "Please update to latest v2.3 or v2.2"
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 727 packages from 824 contributors and audited 740 packages in 33.251s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-07 04:39:00,745 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.0.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.0.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 740 packages in 19.9s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.15.4/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.15.4/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated fsevents@2.1.3: "Please update to latest v2.3 or v2.2"
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 268 packages from 174 contributors and audited 368 packages in 7.212s

82 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-01-07 04:39:29,569 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.0.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14
2021-01-07 04:40:13,112 synthtool [DEBUG] > Compiling protos...
DEBUG:synthtool:Compiling protos...
2021-01-07 04:40:19,204 synthtool [DEBUG] > Post-processing completed
DEBUG:synthtool:Post-processing completed
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "/tmpfs/src/github/synthtool/synthtool/metadata.py", line 250, in __exit__
    self.observer.stop()
AttributeError: 'MetadataTrackerAndWriter' object has no attribute 'observer'
2021-01-07 04:40:21,252 autosynth [ERROR] > Synthesis failed
2021-01-07 04:40:21,252 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bdb71e4 ci: fix bucket_not_found for samples tests (#978)
2021-01-07 04:40:21,283 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-07 04:40:21,310 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/47cb250d-82c0-4285-8db4-42c5067d8370/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

--- a/synth.py
+++ b/synth.py
@@ -24,8 +24,6 @@ logging.basicConfig(level=logging.DEBUG)
 
 AUTOSYNTH_MULTIPLE_COMMITS = True
 
-s.metadata.set_track_obsolete_files(False)
-
 gapic = gcp.GAPICBazel()
 version='v2'
 # tasks has two product names, and a poorly named artman yaml

--- a/synth.py
+++ b/synth.py
@@ -24,7 +24,7 @@ logging.basicConfig(level=logging.DEBUG)
 
 AUTOSYNTH_MULTIPLE_COMMITS = True
 
-s.metadata.set_track_obsolete_files(True)
+s.metadata.set_track_obsolete_files(False)
 
 gapic = gcp.GAPICBazel()
 version='v2'

